interfaces/types.yaml

```diff
@@ -46,6 +46,10 @@ CSTRING:
     # A null-terminated immutable C string
     CTYPE: const char*
 
+CSTRING_P:
```

Comment:
This is not needed; declaring a parameter as `OUT CSTRING` will add an extra level of pointer indirection automatically.